playbook/app/pb_kits/playbook/pb_table/docs/_table_with_collapsible_with_custom_click.jsx

```diff
@@ -46,8 +46,16 @@ return (
             <Table.Cell>{'Value 2'}</Table.Cell>
             <Table.Cell>{'Value 3'}</Table.Cell>
             <Table.Cell>{'Value 4'}</Table.Cell>
-            <Table.Cell>{'Value 5'}</Table.Cell>
-            <Table.Cell cursor="pointer" 
+            <Table.Cell cursor="pointer"
+                id="cell-100"
+            >
+              <Icon
+                  color="primary"
+                  fixedWidth
+                  icon="chevron-down"
```

Comment:
I would use a different icon for this doc example, just to avoid confusion, maybe do something similar to [this](https://pr4582.playbook.beta.px.powerapp.cloud/kits/table/react#table-with-2-actions-1) just in the second last column as opposed to the last one?